server.py

```diff
@@ -345,8 +405,8 @@ async def call_tool(
     async def list_tools() -> list[types.Tool]:
         return [
             types.Tool(
-                name="convert_document",
-                description="Convert a document from a URL or local path to markdown format",
+                name="qna_from_document",
```

Comment:
should this be a separate tool? I.e., people would want to convert_documents to json/markdown but also generate_qna_from_document?

Reply:
They are! The diff is misleading because i put the new tool on top of the list, but the convert tool is still there, just below it.
BTW: Do you think order matters?

Reply:
Ah, thanks!
>BTW: Do you think order matters?

I bleieve so, I think tool descriptions are important too.